content/docs/integrations/kubernetes/index.md

```diff
@@ -104,15 +104,16 @@ You should see a page like this
 
 1. Edit your Kubernetes aws-auth Configmap as per: https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html
 
-It should look something like this:
-``` yaml
- - mapRoles:
-   - "groups":
-      - "system:authenticated"
-      "rolearn": "ROLE ARN FROM STEP TWO"
-      "username": "roadie"
+This is a suggested method using eks:
```

Comment:
I guess the whole set of steps are just suggested steps. Because all of these steps would be done in code whether pulumi, terraform or what have you.

Reply:
Indeed!

Reply:
yeah, would be good to point out what brian said too